hello-world/src/App.js

```diff
@@ -0,0 +1,76 @@
+// import logo from "./logo.svg";
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = JSON.parse(window.localStorage.getItem("voteCount")) || {
+      languages: [
+        { name: "Python", votes: 0 },
+        { name: "Javascript", votes: 0 },
+        { name: "java", votes: 0 },
+        { name: "C++", votes: 0 },
+        { name: "Kotlin", votes: 0 },
+      ],
+    };
+  }
+  static getDerivedStateFromProps(props, state) {
```

Comment:
@shubhamshukla91 
Any specific reason why the lifecycle `static getDerivedStateFromProps(props, state)` has been used here? Especially when the props & state are not being compared to set data to local storage.

`static getDerivedStateFromProps(props, state)` should be used when the internal state of the component needs to be updated on the change of the props it is receiving and then any action is performed using that value in that state.
For example:

```
static getDerivedStateFromProps(props, state) {
        if(props.name !== state.name){
            //Change in props
            return{
                name: props.name
            };
        }
        return null; // No change to state
    }
```
Change the implementation of your code here according to the proper use of this lifecycle. If this lifecycle is not needed, then remove it.